src/kvasir/mpl/algorithm/filter.hpp

```diff
@@ -3,5 +3,20 @@
 //    (See accompanying file LICENSE.md or copy at
 //          http://www.boost.org/LICENSE_1_0.txt)
 #pragma once
+#include "remove_if.hpp"
```

Comment:
cant really factor stuff out of remove if into filter because its used in both places